### PR TITLE
Gloo python bindings

### PR DESCRIPTION
**Pull request by nirandaperera**

### CODE REVIEW

File: python/pycylon/setup.py [hunk 1]
```diff
@@ -151,11 +178,11 @@
...
-
-print('Libraries:', libraries)
-print("Lib dirs:", library_directories)
-print("Include dirs:", _include_dirs)
-print("Macros:", macros)
+print("CYTHON files : " + str(cython_files))
+print('Libraries    :', libraries)
+print("Lib dirs     :", library_directories)
+print("Include dirs :", _include_dirs)
+print("Macros       :", macros)
```
Comment (vibhatha) [inline python/pycylon/setup.py:185]: should we remove these?

File: python/pycylon/setup.py [hunk 2]
```diff
@@ -56,8 +56,34 @@
...
+                'pycylon/data/data_type.pyx',
+                'pycylon/api/lib.pyx',
+                'pycylon/api/types.pyx',
+                'pycylon/indexing/cyindex.pyx',
+                'pycylon/indexing/index_utils.pyx',
+                'pycylon/ctx/context.pyx',
+                'pycylon/io/csv_write_config.pyx',
+                'pycylon/io/csv_read_config.pyx',
+                'pycylon/common/status.pyx',
+                'pycylon/common/join_config.pyx',
```
[the remark this answers is not in the document]
Comment (vibhatha) [reply python/pycylon/setup.py:84]: `pycylon/data/**`, `pycylon/api./**` something like this, but I am not sure, what is the best way to capture the modularized ones. But the existing one is satisfactory for the moment.

**Review by vibhatha — CHANGES_REQUESTED** [review]
NIT

**Comment by nirandaperera** [comment]
@vibhatha this works well in my machine. I dont understand how these changes would affect the runtime path. :slightly_frowning_face:

**Comment by vibhatha** [comment]
@nirandaperera seems CIs are failing.

**Comment by vibhatha** [comment]
> @vibhatha this works well in my machine. I dont understand how these changes would affect the runtime path. 🙁

you mean the changes I suggested? Or the changes we did to include Gloo?

**Comment by nirandaperera** [comment]
> > @vibhatha this works well in my machine. I dont understand how these changes would affect the runtime path. slightly_frowning_face
> 
> you mean the changes I suggested? Or the changes we did to include Gloo?

So, in my local machine, I can get these changes to work without a problem. But somehow, in CI, its failing

**Pull request merged**